### PR TITLE
Major restructure of README, including the addition of new MD files

### PR DESCRIPTION
**Pull request by tphalp**
As I was making the demo, I found some things that could be cleaned up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,142 +4,31 @@ This repo is a monorepo, which contains the pieces that make up the Universal Co
...
-1. Preferences
-1. Institutions
-
-## Getting Started (production)
+## Table of Contents
 
-To get started: clone the repo, follow the steps in [Getting Started](#getting-started-in-development) and
-[Initial Setup](#initial-setup) to set up your `.env` files, and then run the following command from the root of the 
-project:
+- [Getting Started](#getting-started)
```
Comment (wesrisenmay-mx) [inline README.md:9]: This table of contents seems out of date now

Comment (tphalp) [reply README.md:9]: Yep, thanks.

File: README.md [hunk 2]
```diff
@@ -4,142 +4,31 @@ This repo is a monorepo, which contains the pieces that make up the Universal Co
...
+- [Documentation](#documentation)
```
Comment (wesrisenmay-mx) [inline README.md:10]: The main README should point to the new md files you created

File: README.md [hunk 3]
```diff
@@ -4,142 +4,31 @@ This repo is a monorepo, which contains the pieces that make up the Universal Co
...
-resulting in the error. To fix this, you will need to run `docker logout` from your terminal, prior to running `docker compose up`
+Please refer to the [UCP documentation](https://docs.universalconnect.org) (Coming soon) for additional information on how to use the widget.
 
-## Getting Started (development)
+## Getting Started
 
 1. Clone the `ucw-app` repo
 1. Run `npm ci` from the root directory
 1. Run `cp ./.env.example ./.env`
 1. Run `cp ./apps/server/.env.example ./apps/server/.env`
```
Comment (wesrisenmay-mx) [inline README.md:25]: This should mention that they need to fill in their values

File: README.md [hunk 4]
```diff
@@ -4,142 +4,31 @@ This repo is a monorepo, which contains the pieces that make up the Universal Co
...
 1. Run `cp ./apps/server/cachedDefaults/preferences.example.json ./apps/server/cachedDefaults/preferences.json` and modify the preferences
-1. Follow [Initial Setup](#initial-setup) (below) for setting-up some required environment variables
 1. Install [Docker](#docker), a dependency for the institution search feature to work
```
Comment (wesrisenmay-mx) [inline README.md:27]: This docker link isn't relevant now

File: MORE-INFO.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+- [Deploying a Production Instance](#deploying-a-production-instance)
+- [Authentication](#authentication)
+- [API Documentation](#api-documentation)
+- [Cached Data](#cached-data)
+- [Troubleshooting](#troubleshooting)
+- [Redis](#redis)
+- [Architecture Decision Records](#architecture-decision-records)
+- [Monorepo](#monorepo)
+
+## Preferences and Aggregator Credentials
```
Comment (wesrisenmay-mx) [inline MORE-INFO.md:16]: I think this section should live in the main README. npm run dev won't work without it. We should call out that they need to sign up with each provider and get the credentials

File: MORE-INFO.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+Take a look at the [docker-compose.yml](./docker-compose.yml) file for more details on how it is set up.
+
+```
+docker compose up
+```
+
+That's it! If you have questions, please reach out to us.
+
+## Authentication
```
Comment (wesrisenmay-mx) [inline MORE-INFO.md:40]: I think this is worthy of being in the main README as well. It's a big callout.

File: MORE-INFO.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+### Docker Login
+
+If you get an error stating anything like `failed to authorize`, or `failed to fetch oauth token`, or a `401 Unauthorized`,
+you may have logged-in previously with a docker login (via cli), and your auth has expired.
+
+You do not need a docker login to pull the UCW docker images, but if you have stale tokens, docker will try to use them, thus
+resulting in the error. To fix this, you will need to run `docker logout` from your terminal, prior to running `docker compose up`
+
+## Redis
```
Comment (wesrisenmay-mx) [inline MORE-INFO.md:69]: npm run dev now spins up redis, so most of this section is irrelevant. I also ripped out the env=dev thing for redis

File: MORE-INFO.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+You might see an error about failure to connect redis. The widget doesn't rely on redis to start, but some providers logic
+require a redis instance. To fix this error you can either:
+
+- Start a local redis instance. This way it will be available at localhost:6379 and the widget will be able to use it
+- Or in `./apps/server/.env`, set `Env=dev`. When this is done, the redis client will use a local in-memory object to handle
+  the cache, and avoid the error. However, this should only be used for testing. The cached values won't expire and also
+  will be cleared on server restart.
+
+## Architecture decision records
```
Comment (wesrisenmay-mx) [inline MORE-INFO.md:78]: This should probably live in CONTRIBUTING.md

**Comment by wesrisenmay-mx** [comment]
It looks like none of our markdowns point to CONTRIBUTING.md

**Comment by wesrisenmay-mx** [comment]
> It looks like none of our markdowns point to CONTRIBUTING.md

I think the DOCKER.md file has the same issue

**Comment by tphalp** [comment]
> It looks like none of our markdowns point to CONTRIBUTING.md

Fixed. I added a section with links to all the new MD files.

File: apps/server/.env.example [hunk 9]
```diff
@@ -1,25 +1,29 @@
- # Env: The mocked environment will mock out your https calls as defined in mock.js
+# Env: The mocked environment will mock out your https calls as defined in mock.js
 Env=pre
 
-SophtronClientId="" # sophtron.com -> api settings -> direct auth -> UserId
-SophtronClientSecret="" # sophtron.com -> api settings -> direct auth -> AccessKey
```
Comment (tphalp) [inline apps/server/.env.example:5]: I looked through the code, and these variables are not used anywhere, and are duplicates of the other "SOPHTRON*" variables.

Comment (wesrisenmay-mx) [reply apps/server/.env.example:5]: Good find

File: PREFERENCES.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+`defaultProviderVolume` (`{ [key: string]: number }`) - An object of providers, with a number representing a percent of volume. The combined volume across all providers must equal 100.
+
+```
+"defaultProviderVolume": {
+  "mx": 50,
+  "sophtron": 50
+}
+```
+_The total for all providers need to add up to 100_
```
Comment (wesrisenmay-mx) [inline PREFERENCES.md:50]: This is repeated several few lines above

File: README.md [hunk 11]
```diff
@@ -1,145 +1,47 @@
-# Universal Connect Widget (Application)
+# Universal Connect Widget
 
-This repo is a monorepo, which contains the pieces that make up the Universal Connect Application. It is a full-stack
-application which anyone can clone and self-host as a way to serve the connect widget via a url which can then be loaded
-into an iframe.
+This repository is a monorepo, which contains the pieces that make up the Universal Connect Application. It is a full-stack
+application, which anyone can clone and self-host as a way to serve the connect widget via a URL, which can then be loaded
+into an iframe. Due to the nature of the architecture, it is language-agnostic.
```
Comment (wesrisenmay-mx) [inline README.md:5]: I don't think the added sentence is clear or helpful.

**Review by wesrisenmay-mx — DISMISSED** [review]

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**